frontend/src/utilities/useWatchDocs.tsx

```diff
@@ -16,7 +16,7 @@ export const useWatchDocs = (
   const [docs, setDocs] = React.useState<OdhDocument[]>([]);
 
   React.useEffect(() => {
-    let watchHandle: ReturnType<typeof setTimeout>;
+    let watchHandle: ReturnType<typeof setTimeout> | undefined;
```

Comment:
Remove the if condition in the return statement.
```suggestion
    let watchHandle: ReturnType<typeof setTimeout>;
```